src/controllers/tutor.js

```diff
@@ -35,13 +35,26 @@ module.exports = {
   },
   searchForTutor: async (req, res) => {
     const { tutorName } = req.query;
+    let firstName = tutorName.toLowerCase();
+    let lastName = tutorName.toLowerCase();
+
+    if (tutorName.indexOf(' ') !== -1) {
+      [firstName, lastName] = tutorName.toLowerCase().split(' ');
+    }
+
```

Comment:
- one-liner for 38-43: `const [firstName, lastName] = tutorName.toLowerCase().split(' ')`
- avoid using `let`